languages/en.ini

```diff
@@ -811,6 +820,7 @@ Map = "Map"
 Map View = "Map View"
 map_results_label = "At this location:"
 Maps = "Maps"
+Markdown and emoji help = "Markdown and emojis help"
```

Comment:
Is there a reason the key and value differ ("emojis" vs. "emoji")? Maybe it would be better to use a more abstract key, like `markdown_and_emoji_help` or else make the two values match exactly.